experiments/githubcloner.sh

```diff
@@ -0,0 +1,41 @@
+echo 'Enter github repository link'
+read LINK 
```

Comment:
Is this file still needed when you have a copy of it in adding-new-experiments/?